browser/brave_browser_features.cc

```diff
@@ -57,6 +57,10 @@ BASE_FEATURE(kNewAndroidOnboarding,
              base::FEATURE_DISABLED_BY_DEFAULT);
 #endif  // BUILDFLAG(IS_ANDROID)
 
+BASE_FEATURE(kBraveEmailAliases,
```

Comment:
please use consistent naming. Sometimes you have email aliases sometimes brave email aliases